### DISCUSSION TITLE
Add to pytest core different collect outputs

### DISCUSSION THREAD

**Discussion by LaserPhaser** [pytest-dev/pytest#8066] [Show and tell] [answered]
I wrote such plugin for my personal needs, 
Maybe we should integrate it to the core 
https://github.com/LaserPhaser/pytest-collect-formatter

**Comment by RonnyPfannschmidt** [comment, marked as answer]
i don't quite see what type of problem this solves

at first glance its just the serialization of nodes in a completely alien, not integrated text format

please provide an example where this adds value/integrates as  a starting point

**Reply by LaserPhaser** [reply]
It's just a question for now, do we need it in the pytest or now. 
From my perspective most of users prefer json or yaml format instead of xml format. 
If you think that we should not integrate it to the pytest-core we could easily left it as a plugin.

BTW what do you mean by "alien, not integrated text format". ?

**Reply by The-Compiler** [reply]
We typically add things to pytest core when we see it's something almost everyone uses or benefits from - for example, that's what's happened with the logging integration or faulthandler plugin (showing better information with low-level crashes).

I don't think there's a good reason to integrate a brand-new plugin with an arguably niche use-case and a small userbase into the core.

**Reply by LaserPhaser** [reply]
Ok got it, thank you for answering.
